flexmeasures/data/schemas/scheduling/process.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import annotations
+
+from datetime import datetime
+import pytz
+import pandas as pd
+
+from marshmallow import (
+    Schema,
+    post_load,
+    fields,
+    pre_load,
+)
+
+from flexmeasures.data.models.time_series import Sensor
+from flexmeasures.data.schemas.times import (
+    DurationField,
+    TimeIntervalSchema,
+)
+
+
+from enum import Enum
+
+
+class ProcessType(Enum):
+    INFLEXIBLE = "INFLEXIBLE"
+    BREAKABLE = "BREAKABLE"
+    SHIFTABLE = "SHIFTABLE"
+
+
+class OptimizationSense(Enum):
+    MAX = "MAX"
+    MIN = "MIN"
+
+
+class ProcessSchedulerFlexModelSchema(Schema):
+    # time that the process last.
+    duration = DurationField(required=True)
+    # nominal power of the process.
+    power = fields.Float(required=True)
+    # policy to schedule a process: INFLEXIBLE, SHIFTABLE, BREAKABLE
+    process_type = fields.Enum(
```

Comment:
Can you tell me if this is case-sensitive?
If yes, we're fine functionality-wise.

Otherwise, or in any case, we can add consistency. For instance, I'm seeing the changelog mention "breakable", the docstrings mention "Breakable" and the CLI using "BREAKABLE".

Reply:
Yes, it's case sensitive. I've harmonized the capitalization!